### PR TITLE
added new attributeMeta type, 'alphaNumeric'

### PR DESCRIPTION
**Pull request by clintropolis**
triggers alphaNumeric metric type sorting in druid

### CODE REVIEW

File: test/unit/driver/druidDriver.mocha.coffee [hunk 1]
```diff
@@ -1413,3 +1414,42 @@ describe "Druid driver", ->
...
+          direction: 'ascending'
+        },
+        limit: 10
+      }))
+
+      query = queryBuilder.getQuery()
+
+      expect(query.metric).to.deep.equal({ type: 'alphaNumeric' })
+
+      testComplete()
```
Comment (clintropolis) [inline test/unit/driver/druidDriver.mocha.coffee]: Any opinions on if this is the correct file for this test?

**Comment by clintropolis** [comment]
Should `matrix` type combines support this sorting?

File: src/driver/druidDriver.ts [hunk 2]
```diff
@@ -712,30 +712,31 @@ module FacetLegacy {
...
+              var metaType = this.getAttributeMeta(this.dimension.dimension).type;
+              if (metaType === "large") {
                 this.context.doAggregateTopNMetricFirst = true;
               }
 
               if (sort.prop === this.dimension.outputName) {
                 if (sort.direction === "ascending") {
                   this.metric = {
-                    type: "lexicographic"
+                    type: metaType == 'alphaNumeric' ? 'alphaNumeric' : "lexicographic"
```
Comment (sebastien-fragnaud) [inline src/driver/druidDriver.ts]: Hmmmm ternaries...

Comment (aprilrd) [inline src/driver/druidDriver.ts]: and triple equals?

File: test/unit/driver/druidDriver.mocha.coffee (hunk 1 above)
Comment (aprilrd) [inline test/unit/driver/druidDriver.mocha.coffee]: Yeah I think it makes sense.

**Comment by aprilrd** [comment]
looks good to me overall except the double equals.

**Comment by aprilrd** [comment]
:stars:

**Pull request merged**